### DISCUSSION TITLE
Integrate GPT-3 for better guessing the input and props

### DISCUSSION THREAD

**Discussion by Hamoonist** [fwouts/previewjs#1323] [Ideas]
Hi, I think if you use GPT-3 API for mock data, it will be awesome.
So, you send the component to the API, and ask it to make some mock data and props for it.
It usually gives very good responses.

**Comment by fwouts** [comment]
I remember seeing this and thought I had replied to it already, but FYI this is on the roadmap. In particular, there will be a button "Generate stories" that will generate a bunch of different stories for you automagically.
